### DISCUSSION TITLE
List of Favourites

### DISCUSSION THREAD

**Discussion by Man-Carrasco** [hugoattal/OrbitalMarket#40] [Q&A]
Hi, I was wondering how do you save the list of favourite assets? I am having issues to see the list on different computers and I do not know what is missing.

**Comment by hugoattal** [comment]
Hey! It's saved in the local storage of your browser. So one browser = one favlist.

It would be better to have an account system so that favlist would be accessible from another computer, but I don't have the bandwidth to add such feature for now 😕...

**Comment by hugoattal** [comment]
You can now import/export favlist if you want to share your favlsit between different computers: https://github.com/hugoattal/OrbitalMarket/discussions/43

I know it's not ideal compared to an account system, but I hope that's still an improvement 😅
